interfaces/cython/cantera/test/utilities.py

```diff
@@ -117,12 +129,12 @@ def compareProfiles(reference, sample, rtol=1e-5, atol=1e-12, xtol=1e-5):
     If the comparison succeeds, this function returns `None`. If the comparison
     fails, a formatted report of the differing elements is returned.
     """
-    if isinstance(reference, str):
+    if isinstance(reference, (str, Path)):
         reference = np.genfromtxt(reference, delimiter=',').T
     else:
         reference = np.asarray(reference).T
 
-    if isinstance(sample, str):
+    if isinstance(sample, (str, Path)):
```

Comment:
I think these should both be `PurePath` as well.

Reply:
This seems like it got missed along the way.

Reply:
Not sure - from what I can tell, these are already changed to `PurePath`?

Reply:
Whoops thanks!